roles/mysql/tasks/main.yml

```diff
@@ -30,8 +35,28 @@
     update_cache: true
   when:
     - mysql_server
-  notify:
-    - set root password
+
+- name: MySQL | Restart MySQL
+  ansible.builtin.service:
+    name: mysql
+    state: restarted
+  when: install_app_key.changed
+
+- name: MySQL | mysql_root_password
+  community.mysql.mysql_user:
+    login_unix_socket: "{{ mysql_socket }}"
+    login_user: root
+    login_host: localhost
+    login_password: ""
+    name: 'root'
+    password: "{{ mysql_root_password }}"
+  when: install_app_key.changed
+
+- name: MySQL | remove unix_plugin
```

Comment:
This isn't a comment on the playbook, so much as a question about our procedure here. What is the unix plugin and why are we installing and then removing it?

Reply:
Mysql password's changed at 5.7 Otherwise it fails. https://mariadb.com/kb/en/authentication-plugin-unix-socket/